includes/controls/groups/grid-container.php

```diff
@@ -149,19 +149,18 @@ protected function init_fields() {
 			'label' => esc_html_x( 'Justify Content', 'Grid Container Control', 'elementor' ),
 			'type' => Controls_Manager::CHOOSE,
 			'label_block' => true,
-			'default' => '',
 			'options' => [
 				'start' => [
 					'title' => esc_html_x( 'Start', 'Grid Container Control', 'elementor' ),
-					'icon' => 'eicon-flex eicon-justify-start-h',
+					'icon' => 'eicon-flex eicon-justify-start-v',
```

Comment:
Unrelated fix. 
The horizontal and vertical icons were swopped.

Reply:
Please remove all the icon fix related code from this PR.
Aviv will ask Louis to add it to [his PR ](https://elementor.atlassian.net/browse/ED-9891)

Reply:
Updated